sdf/whatsnew.mdx

```diff
@@ -6,9 +6,44 @@ sidebar_position: 20
 
 # Beta 2
 
+## Upgrading 
+
+To upgrade to the beta2, run the following command:
+
+```bash
+$ fvm install sdf-beta2
+```
+
+For upgrading cloud workers, please contact InfinyOn support. 
+
+### Compatibility and Breaking changes
+
+V4 of dataflow is no longer supported. Please upgrade to V5.
+Beta2 CLI is not backward compatible with Beta1 workers.  Please upgrade workers to Beta2 in order to use the Beta2 CLI.
+
+### CLI changes
+
+publishing:
+`sdf hub publish` -> `sdf publish`
+
+deployment changes, see [deployment]:
+`sdf run` -> `sdf deploy`,  
```

Comment:
is there a link to the new CLI command that explains how these commands are different?

Reply:
it's in the deployment link